lambdatimes/src/components/Header.js

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
-
+import styled from "styled-components"
 // Refactor this component to use styled components and not classNames. 
 // You can find the corresponding CSS in the CSS/index.css file
 
 const Header = () => {
   return (
-    <div className="header">
-      <span className="date">SMARCH 32, 2018</span>
-      <h1>Lambda Times</h1>
-      <span className="temp">98°</span>
-    </div>
+    <HeaderDiv className="header">
```

Comment:
Looks like you're starting to build out the styled components, just make sure you define them in this file!